frontend/src/pages/Users/Users.tsx

```diff
@@ -104,10 +104,7 @@ export const Users: React.FC = () => {
         row.full_name = `${row.first_name} ${row.last_name}`;
       });
 
```

Comment:
Add a Playwright spec that:
- Logs in as Regional Admin
- Tries to hit the Admin Tools path by URL
- Expects redirect or 403 page

Files:
frontend/src/pages/Users/Users.tsx and related guards
A Playwright spec under playwright/e2e/role-regional-admin.spec.ts confirming the bounce

Reply:
A ticket was created to make playwright tests specifically for Regional Admin access. I've added a comment to the ticket for a request to make this specific test here: https://maestro.dhs.gov/jira/browse/CRASM-3183

Reply:
As discussed, the new ticket depends on this one as this updates the regional admin permissions. A TODO comment has been added here: [e3323b8](https://github.com/cisagov/XFD/pull/1283/commits/e3323b8ee080595b98dbe3ed1df0e18855d3a9dd)